release/build_release.sh

```diff
@@ -106,6 +106,35 @@ function cherry_pick_commits {
 	git push origin -q -u "${LIFERAY_RELEASE_GIT_REF}"
 }
 
+function create_branch {
+	local original_branch_name=""
```

Comment:
@kiwm, please set `local original_branch_name="master"`. This way, you may remove the `else` below.